datahub-web-react/src/app/shared/tags/TagProfileDrawer.tsx

```diff
@@ -0,0 +1,417 @@
+    }
+`;
+
+const DescriptionText = styled(MarkdownViewer)`
+    padding-right: 8px;
+    display: block;
+`;
+
+const EditIcon = styled(EditOutlined)`
+    cursor: pointer;
+`;
+
+const EditDescriptionText = styled(Typography.Text)`
+    &&& {
+        font-size: 13px;
+        font-weight: 500;
+    }
+`;
+
+const HeaderLayout = styled.div`
+    display: flex;
+    justify-content: space-between;
+`;
+
+const StatsBox = styled.div`
+    width: 180px;
+    justify-content: left;
+`;
+
+const StatsLabel = styled(Typography.Text)`
+    &&& {
+        font-size: 13px;
+        font-weight: 600;
+    }
+`;
+
+const EmptyStatsText = styled(Typography.Text)`
+    font-size: 15px;
+    font-style: italic;
+`;
+
+const StatsButton = styled(Button)`
+    padding: 5.6px 0px;
+    margin-top: -4px;
+`;
+
+const ExpandedActions = styled.div`
+    height: 10px;
+`;
+
+const ReadLessText = styled(Typography.Link)`
+    margin-right: 4px;
+`;
+
+const ABBREVIATED_LIMIT = 80;
+
+export const TagProfileDrawer = ({ closeTagProfileDrawer, tagProfileDrawerVisible, urn }: Props) => {
+    const { loading, error, data, refetch } = useGetTagWithColorQuery({ variables: { urn } });
+    const entityRegistry = useEntityRegistry();
+    const history = useHistory();
+    const [updateDescription] = useUpdateDescriptionMutation();
+    const [setTagColorMutation] = useSetTagColorMutation();
+
+    const entityAndSchemaQuery = `tags:"${data?.tag?.properties?.name}" OR fieldTags:"${data?.tag?.properties?.name}" OR editedFieldTags:"${data?.tag?.properties?.name}"`;
+    const entityQuery = `tags:"${data?.tag?.properties?.name}"`;
+    const ownersEmpty = !data?.tag?.ownership?.owners?.length;
+    const description = data?.tag?.properties?.description || '';
+    const hexColor = data?.tag?.properties?.colorHex || '';
+
+    const [showAddModal, setShowAddModal] = useState(false);
+    const [showEditModal, setShowEditModal] = useState(false);
+    const [updatedDescription, setUpdatedDescription] = useState('');
+    const overLimit = removeMarkdown(updatedDescription).length > 80;
+    const [expanded, setExpanded] = useState(!overLimit);
+    const [displayColorPicker, setDisplayColorPicker] = useState(false);
+    const [colorValue, setColorValue] = useState('');
+
+    useEffect(() => {
+        setUpdatedDescription(description);
+        // Manual Color Set-Up, Need to set once it will come from backend
+        setColorValue(hexColor);
+    }, [description, hexColor]);
+
+    const allSearchResultsByType = useGetAllEntitySearchResults({
+        query: entityAndSchemaQuery,
+        start: 0,
+        count: 1,
+        filters: [],
+    });
+
+    const statsLoading = Object.keys(allSearchResultsByType).some((type) => {
+        return allSearchResultsByType[type].loading;
+    });
+
+    const someStats =
+        !statsLoading &&
+        Object.keys(allSearchResultsByType).some((type) => {
+            return allSearchResultsByType[type]?.data?.search.total > 0;
```

Comment:
while you're at it, can you switch this to using the all entities search? For example of usage: https://github.com/linkedin/datahub/blob/master/datahub-web-react/src/app/entity/shared/components/styled/search/EmbeddedListSearch.tsx#L77. This only has 1 API call total while this deprecated approach has many.